spl-balance/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "spl-balance"
+version = "0.1.0"
+description = "Messari's standardized substream for solana SPL tokens"
+edition = "2021"
+repository = "https://github.com/messari/substreams"
```

Comment:
Not really that important but it would be nice to have all these links be consistant. (If you use the CLI to create the substream projects it will do this for you :) )